### ISSUE TITLE
Add Network AP Config

### ISSUE THREAD

**Issue by Anti-T97** [vgavro/samsung-mdc#14]
Hi Victor, 

I have another request 
i try to add code for add SSID and password
![image](https://github.com/user-attachments/assets/5d458484-e193-45c8-b6bb-730670fb2bef)

I add this code : 
```
class NETWORK_AP_CONFIG(Command):
    CMD = 0x1B
    SUBCMD = 0x8A
    GET, SET = False, True

    DATA = [Str('SSID'), Str('PASSWORD_SSID')]

```
It not realy work fine...

Possible to help me to add it ?

Thank

**Comment by Anti-T97** [comment]
OK Thank you
i try it today

**Comment by vgavro** [comment]
Can you try to increase timeout? To some excessive number like 30s?
All mdc commands I've seen before always sending acknowledge. So - either this command does not. Either it tries to validate before sending acknowledge.

Also. What about other commands. Especially current time.

**Comment by Anti-T97** [comment]
i have right
when i send the command connection time out since 20s
i retry with delay

**Comment by vgavro** [comment]
Can you plz *explicitly* confirm that on this version (from branch above):

1. command status is working (get only)  (if not - full error and was it working on previous version)
2. command clock_s is working  (get only) (if not - full error and was it working on previous version)
3. command panel_on_time is working  (get only) (if not - full error and was it working on previous version)
4. command rgb is working (get only) (if not - full error and was it working on previous version)
5. command model_name is working (get only) (if not - full error and was it working on previous version)

6. do I understand correctly that EACH time NETWORK_AP_CONFIG command is applied with SUCCESS (proper params), *device* is changing network? IF so - maybe this should be added to command readme?

7.1. if 6. is true - what about applying same SSID,PASSWORD? it shouldn't change network if it's already on this network, right? so it should return expected result?

7.2. If you meant YOUR (client) network is changed (for some reason) - could you avoid this and confirm command is working properly when your network doesn't change?

8. in any case - applying wrong credentials shouldn't change device device network? so what is returned then - error or success?

**Comment by Anti-T97** [comment]
1. Status OK : 0@192.168.1.66 <POWER_STATE.ON:1> 0 <MUTE_STATE.OFF:0> <INPUT_SOURCE_STATE.MEDIA_MAGIC_INFO_S:96> <PICTURE_ASPECT_STATE.PC_16_9:16> 0 0
2. Clock_s OK : 0@192.168.1.66 2024-07-22T14:45:37
3. Panel_on_time Error : 0@192.168.1.66 MDCResponseError: ('Unparsed data left', b'y')
4. RGB OK : 0@192.168.1.66 70 45 <COLOR_TONE_STATE.OFF:80> 100 0 255 255 255
5. Model_name OK : 0@192.168.1.66 QM55B-T
6. Exact, when it change network the PC sent the command it on another after command success
7.1. When set the same SSID and PASS on the same network the timeout is to short for get the response.
0@192.168.1.66 MDCResponseError: ('Empty response', b'')
7.2. I need to change SSID on screen for change Network not juste for change SSID on the same Network.
8. Wrong Credential disconnect to the current SSID or Cable Network, need to manually reconnect with the remote control

**Comment by vgavro** [comment]
thanks for your tests! Looks like everything is working as expected.

I added note to NETWORK_AP_CONFIG about network change, in protocol they explicitly state that there should be response, so let's consider this behaviour as *device-specific* (not *protocol-specific*) and leave it on "notes" level in command doc.

About `panel_on_time` error - I added unit tests, and it looks like command is working properly (regarding protocol), i believe either you have newer device which behaves ondocumented... If you have time - plz run command with `--verbose` option to see actual response, i believe if you compare it to protocol - response contains more data then it should. Plz create new ticket on this, i'm closing this ticket and merging changes to new branch. Thanks for help!
